tests/validation/cattlevalidationtest/core/test_services.py

```diff
@@ -556,22 +558,26 @@ def test_service_reconcile_delete_instance_restart_policy_always(
     service, env = create_env_and_svc_activate_launch_config(
         super_client, client, launch_config, scale)
     check_for_service_reconciliation_on_delete(super_client, client, service)
+    delete_all(client, [env])
 
 
-def test_service_reconcile_stop_instance_restart_policy_no(
+def test_service_reconcile_delete_instance_restart_policy_no(
         super_client, client, socat_containers):
     scale = 3
     launch_config = {"imageUuid": TEST_IMAGE_UUID,
-                     "restartPolicy": {"name": "no"}}
+                     "labels": {"io.rancher.container.start_once": True}
+                     }
     service, env = create_env_and_svc_activate_launch_config(
         super_client, client, launch_config, scale)
+    check_for_service_reconciliation_on_delete(super_client, client, service)
+    delete_all(client, [env])
 
 
-def test_service_reconcile_delete_instance_restart_policy_no(
+def test_service_reconcile_stop_instance_restart_policy_no(
         super_client, client, socat_containers):
     scale = 3
     launch_config = {"imageUuid": TEST_IMAGE_UUID,
-                     "restartPolicy": {"name": "no"}}
+                     "labels": {"io.rancher.container.start_once": True}}
```

Comment:
I'm not totally sure that `True` will work.  Does the API turn that in into `"true"`?